### DISCUSSION TITLE
Why can't my compiled exe run in windows server 2012R2 system

### DISCUSSION THREAD

**Discussion by J1nAB1n9** [charmbracelet/bubbletea#666] [Q&A]
Why can't my compiled exe run in windows server 2012R2 system
![T_H H1L9 M{~M~E1YK_ZT2X](https://user-images.githubusercontent.com/42513580/201821111-afacd156-af4b-49af-87e5-beb660b0271a.png)

**Comment by tsal** [comment]
@J1nAB1n9 - this would be because the "command prompt" terminal emulator on older versions of windows does not support the control codes needed for console apps to work. That is, the windows "command prompt" (or even older powershell terminals) isn't a fully-functional console.  There is a windows terminal app you can now get from Microsoft that should support the control codes.
